package.json

```diff
@@ -78,6 +78,7 @@
     "react-dom": "0.14.7",
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
+    "redux-thunk": "^2.1.0",
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/376/commits/77e6c33e877d6931f17e4958e855171bc1cf814b